dataeng/jobs/analytics/SnowflakeExpirePasswords.groovy

```diff
@@ -0,0 +1,88 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+
+
+
+class SnowflakeExpirePasswords {
+
+    public static def job = { dslFactory, allVars ->
+
+        Map SnowflakeExpirePasswordsQuarterlyConfig = [
+            NAME: 'expire-snowflake-passwords-quarterly',
+            MANUAL: false,
+            SCRIPT_TO_RUN: 'dataeng/resources/snowflake-expire-passwords.sh'
+        ]
+        Map SnowflakeExpirePasswordsManuallyConfig = [
+            NAME: 'expire-snowflake-passwords-manually',
+            MANUAL: true,
+            SCRIPT_TO_RUN: 'dataeng/resources/snowflake-expire-individual-password.sh'
+        ]
+        List jobConfigs = [
+            SnowflakeExpirePasswordsQuarterlyConfig,
+            SnowflakeExpirePasswordsManuallyConfig
+        ]
+
+        jobConfigs.each { jobConfig ->
+
+            dslFactory.job(jobConfig['NAME']) {
+
+                logRotator common_log_rotator(allVars)
+                parameters secure_scm_parameters(allVars)
```

Comment:
This job doesn't make use of secure scm parameters. We can get rid of `secure_scm_parameters` and `secure_scm` down below.